package.json

```diff
@@ -149,17 +149,16 @@
 		"react-docgen-typescript-loader": "^3.3.0",
 		"react-dom": "^16.6.3",
 		"sinon": "^4.5.0",
-		"storybook-chromatic": "^3.5.5",
```

Comment:
Was this removed on purpose?

Reply:
Yeah. We replaced it with the "chromatic" package which is more up to date and does the same thing.